services/discovery/tests/unit/test_handler.py

```diff
@@ -349,15 +357,16 @@ def test_handler_non_master_payer_invalid(context, cfn_event, describe_trails_re
         'IsAuditAccount': True,
         'IsCloudTrailOwnerAccount': True,
         'IsResourceOwnerAccount': True,
-        'IsMasterPayerAccount': False,
+        'IsMasterPayerAccount': True,
         'IsOrganizationMasterAccount': False,
         'MasterPayerBillingBucketName': None,
         'MasterPayerBillingBucketPath': None,
     }
 
 
+# This is actually not possible
 @pytest.mark.unit
-def test_handler_non_master_payer_remote(context, cfn_event, describe_trails_response_local, list_buckets_response, describe_report_definitions_response_remote, describe_organizations_remote):
+def test_handler_master_payer_remote(context, cfn_event, describe_trails_response_local, list_buckets_response, describe_report_definitions_response_remote, describe_organizations_remote):
```

Comment:
I kept this test to ensure the code returns `None` for `MasterPayerBillingBucketName` and `MasterPayerBillingBucketPath`. In this case, the downstream CFN will create the report.